fusor-ember-cli/app/routes/subscriptions/credentials.js

```diff
@@ -3,39 +3,27 @@ import request from 'ic-ajax';
 
 export default Ember.Route.extend({
 
-  setupController(controller, model) {
-    controller.set('model', model);
-    controller.set('showErrorMessage', false);
-
-    // check if org has upstream UUID using Katello V2 API
-    var orgID = this.modelFor('deployment').get('organization.id');
-    var url = '/katello/api/v2/organizations/' + orgID;
-    Ember.$.getJSON(url).then(function(results) {
```

Comment:
Refactoring. This code isn't needed since it's a duplicate after it runs in the parent route 'deployment'